app/src/main/res/layout/article_fragment.xml

```diff
@@ -1,15 +1,79 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/article_fragment_id"
     android:layout_width="match_parent"
-    android:background="#fff"
     android:layout_height="match_parent"
-    android:padding="16dp">
+    android:background="#eee">
+
+    <!-- A CardView that contains a TextView -->
+    <android.support.v7.widget.CardView xmlns:card_view="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/card_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="2dp"
+        android:background="?android:attr/selectableItemBackground"
+        android:minHeight="?android:attr/listPreferredItemHeightSmall"
+        card_view:cardCornerRadius="6dp"
+        card_view:cardElevation="4dp"
+        card_view:cardUseCompatPadding="true"
+        card_view:contentPaddingBottom="2dp">
+
```

Comment:
I tend to avoid Relative layouts if I can, they tend to get messy and error-prone. LinearLayout for president